proto/keystore_api/v1/keystore.proto

```diff
@@ -178,8 +158,17 @@ message CreateAuthTokenRequest {
     optional uint64 timestamp_ns = 1;
 }
 
+// SaveV1ConversationsRequest is used to save a batch of conversations to the
+// built in persistence
+message SaveV1ConversationsRequest {
+    repeated xmtp.message_contents.ConversationReference conversations = 1;
+}
+
+// Placeholder response type for SaveV1Conversations
+message SaveV1ConversationsResponse {}
+
 // Response for GetV2Conversations
-message GetV2ConversationsResponse {
```

Comment:
This existing proto was unused, but will be used once the Snap code gets merged into `beta`. Made the name generic to handle both V1 and V2 convos.